index.html

```diff
@@ -0,0 +1,216 @@
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Github </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Codepen </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Gitlab </p>
+                    </li>
+                    <li class="textPlaceholder element">
+                        <p> Terminal </p>
+                    </li>
+                </ul>
+            </li>
+        </ul>
+    </section>
+    <section id="contact" class="container">
+        <div class="content contact contact-parent">
+            <p> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+            <form class="contact" action="https://formspree.io/f/xqknzgnq" method="post">
+                <label for="name">
+                    <input name="user-name" type="text" id="name" placeholder="Full name" maxlength="30" required>
+                </label>
+                <label for="email">
+                    <input name="user-email" type="email" id="email" placeholder="Email address">
+                </label>
+                <label for="message">
+                    <textarea name="comment" id="message" maxlength="500" placeholder="Enter text here"
+                    required></textarea>
+                </label>
+                <button type="submit" class="interaction">Get in touch</button>
+                <span class="error"></span>
+            </form>
+        </div>
+    </section>
+    <footer class="contact">
+        <ul class="container socialGroup">
+            <li class="social-media">
+                <a rel="noopener noreferrer" href="https://github.com/ssenkooto1" target="_blank"
+                    aria-label="GitHub"><img src="icons/blacat.svg" alt="Github.com Icon"></a>
+            </li>
+            <li class="social-media">
+                <a rel="noopener noreferrer" href="https://www.linkedin.com/in/stephen-ssenkooto-00a036b1/"
+                    target="_blank" aria-label="LinkedIn"><img src="icons/blackIN.svg" alt="LinkedIn.com Icon"></a>
+            </li>
+            <li class="social-media">
+                <a rel="noopener noreferrer" href="#" aria-label="Angelist"><img src="icons/peace.svg"
+                        alt="Angelist.com Icon"></a>
+            </li>
+            <li class="social-media">
+                <a rel="noopener noreferrer" href="https://twitter.com/SsenkootoSteph1" target="_blank"
+                    aria-label="Twitter"><img src="icons/blacktwitter.svg" alt="Twitter.com Icon"></a>
+            </li>
+            <li class="social-media">
+                <a rel="noopener noreferrer" href="#" aria-label="Medium"><img src="icons/blackM.svg"
+                        alt="Medium.com Icon"></a>
+            </li>
+        </ul>
+    </footer>
```

Comment:
- [ ] [OPTIONAL] Good you added these section. But the following lines of codes will be required later on in the portfolio application development.
Implementing them now in this specific task is not a good idea, please try not to go beyond the task requirements.